definitions/mlops-machine_learning_model/definition.yml

```diff
@@ -17,12 +17,29 @@ synthesis:
           entityTagName: providerExternalId
         aws.sagemaker.endpoints.modelmetrics.MonitoringSchedule:
           entityTagName: monitoringSchedules
+    - identifier: modelName
+      name: modelName
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: metricType
+          value: model_metric
+    - identifier: modelName
+      name: modelName
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: metricType
+          value: data_metric
+    - identifier: modelName
+      name: modelName
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: eventType
+          value: InferenceData
 dashboardTemplates:
-  newRelic:
+  AWS:
```

Comment:
This is the `instrumentation.provider` tag. Is the tag being provided from all rules?

Have also all the rules the same data shape or we need to provide a dashboard per each?

Reply:
the tag is being provided from all types of data
the dashboard, for now, is relevant just for AWS, we're working on developing our own screens for this entity and we want the data to go into the entity in order to precede the development process